app/packages/multimodal/src/adapters/mcap/react/McapTile.settings.module.css

```
@@ -10,6 +10,25 @@
+.growField {
+  flex: 1;
+  min-height: 0;
+}
+
+.growPre {
+  flex: 1;
+  margin: 0;
+  min-height: 120px;
+  overflow: auto;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)